econml/tests/test_two_stage_least_squares.py

```diff
@@ -40,7 +40,7 @@ def test_hermite_results(self):
 
     @pytest.mark.slow
     def test_hermite_approx(self):
-        n = 500000
+        n = 50000
```

Comment:
This test was occasionally causing failures because it would exhaust the memory of the build machines, so I've reduced the number of rows.